docs/src/selectors.md

```diff
@@ -284,6 +284,61 @@ converts `'//html/body'` to `'xpath=//html/body'`.
 
 Attribute engines are selecting based on the corresponding attribute value. For example: `data-test-id=foo` is equivalent to `css=[data-test-id="foo"]`, and `id:light=foo` is equivalent to `css:light=[id="foo"]`.
 
+## Selecting element from the list
```

Comment:
I find this title misleading - selecting element from the list can be done with nth-child. You should name it "Pich n-th match from the query result" or something like this.